Makefile

```diff
@@ -20,9 +20,14 @@ run: ## Run via godep (without building)
 	go run *.go
 
 all: test build docker ## Test, build and docker image build
+	
+fmt: ## Run go fmt for all files in the project (excluding vendor)
+	go fmt $(shell go list ./... | grep -v /vendor/)
 
-test: ## Perform both unit and integration tests
-	go test -v -cover -tags "unit integration" $(TEST_PACKAGES)
```

Comment:
Why is separated the `tests` as `unit tests` `integration tests` tags? in `go` all the test end with `_test.go` , for me the differentiation of the test belongs to the folder they live(for ex: folder `database - integration`, folder `domain/model-unit`). 
Anyway you could do the same here with :  
```shelll 
go test ./... -v -cover
```